guides/common/modules/proc_generating-a-list-of-installed-packages.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Generating_a_List_of_Installed_Packages_{context}"]
+= Generating a List of Installed Packages
+
+Use this procedure to generate a list of installed packages in *Report Templates*.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Monitor* > *Report Templates*.
+. Select the *Host - All Installed Packages* report template.
+
+Under the *Editor* tab, the report template shows the host’s name with the name of the package.
```

Comment:
Isn't it the **Preview** tab?

Reply:
Could you please verify with SME?
Unless it changed from the last the the testing instance was updated, this is what editor tab curretly shows. I can't quite make out host name or package name from that. Compare to screenshot from preview tab where host names and package names are present.
### Editor
![image](https://user-images.githubusercontent.com/108661422/219410502-68bed565-d07b-4562-8489-760bb829ccb3.png)
### Preview
![image](https://user-images.githubusercontent.com/108661422/219410642-8908d3fd-91e8-4867-9edd-796bc9d9a0e4.png)

Reply:
I'll verify with the SME.

Reply:
You do need to switch to the Preview tab if you want to preview changes to the report templates.  That only applies if you want to edit it though. Otherwise, users can stick to just generating it.

Reply:
Thanks for the clarification!